### ISSUE TITLE
No puedo hacer Push

### ISSUE THREAD

**Issue by NicolasTheHuman** [teknomanBlade/AM2_TP2#5]
Me tira error cada vez que quiero hacer un push

![image](https://user-images.githubusercontent.com/56463262/97378988-79abc180-18a2-11eb-8726-a0a2abac919f.png)

**Comment by teknomanBlade** [comment]
Fijate de loguearte desde la pagina de GitHub de vuelta por que tenés que refrescar el Token de seguridad para acceder. Cuando te loguees de vuelta te va a pedir la pass probablemente y un codigo a tu mail.

**Comment by EmilianoGaeta** [comment]
Proba esto https://es.stackoverflow.com/questions/300905/como-usar-otra-cuenta-de-github-en-windows
quizás tengas asociada a windows otra cuenta

**Comment by NicolasTheHuman** [comment]
Hola
Me fije en las credenciales y las tengo bien, no tengo otra cuenta de github.

![image](https://user-images.githubusercontent.com/56463262/97593708-b54ca580-19e0-11eb-80d4-eb19a3ed32b2.png)

**Comment by EmilianoGaeta** [comment]
Podes probar pusheando en otro proyecto? create uno y fíjate si anda bien.

**Comment by EmilianoGaeta** [comment]
Proba una cosa en ese mismo lugar donde tenes las credenciales de windows quita todo lo de github, apreta "Remove", luego volve a loguearte en GithubDesktop, asi se vuelve a generar el credencial

**Comment by NicolasTheHuman** [comment]
Si hago un repositor propio me deja pushear.

Removí las credenciales y el problema persiste.

**Comment by EmilianoGaeta** [comment]
Si quitaste todas las credenciales, te deslogueaste y te volviste a loguear, eso es lo mas importante, y todavia no anda.
Ahora probaría que se fijen si estas realmente como colaborador.

**Comment by NicolasTheHuman** [comment]
Sí, me dosloguee desde el github desktop y de la página, y no se me arregló.

**Comment by teknomanBlade** [comment]
![image](https://user-images.githubusercontent.com/5577952/97629211-8a784680-1a0c-11eb-8060-513574958537.png)

Les paso la imagen donde figura como colaborador Nicolas.

**Comment by EmilianoGaeta** [comment]
Proba agregar un archivo directamente en github, literalmente podes arrastrar un archivo al proyecto, mira si te deja subir asi.
Despues probaria desinstalar githubdesktop, y reinstalarlo haciendo nuevamente lo de las credenciales.

Seguire investigando.

**Comment by teknomanBlade** [comment]
Buenas. Ahi pude ver que logró pushear algo. Veo el commit como el primero que hacer en la parte de Contributors.

**Comment by NicolasTheHuman** [comment]
Sí, desde github añadí un documento de texto

**Comment by EmilianoGaeta** [comment]
Podes probar desde otra computadora, asi vemos si el problema esta en tu pc.

**Comment by NicolasTheHuman** [comment]
Pude hacer un push desde la PC de mi hermano con mi cuenta, así que el problema esta en la PC
